possel/data/frontend/README.md

```diff
@@ -0,0 +1,18 @@
+This readme is to help those working on the frontend.
+
+There are several dependencies that need to be included.
+
+They can be installed as followed:
+
+    gem install jekyll
+    npm install
```

Comment:
It was the npm line that was a problem (reviewninja has an off by one bug on line comments)